benchmarks/octane/octane.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const fs = require('fs');
+const vm = require('vm');
+
+global.load = load;
+global.read = read;
+global.print = console.log;
+
+process.chdir('octane')
```

Comment:
Ah, okay.  In that case you may want to write it as `process.chdir(__dirname + '/octane')` to make it independent of the current working directory.